### PR TITLE
Update: delete old profile & update new profile

### PR DESCRIPTION
**Pull request by imaaquibali**
## Changes proposed

- I deleted my old profile aaquibdev.json
- Update the community discord name

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5095"></a>

### CODE REVIEW

**Review by himanshu1221 — APPROVED** [review]
**LGTM :+1:**

File: data/imaaquibali.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     },
     {
       "group": "The VerSiton Community",
-      "name": "EddieHub: Discord",
```
Comment (WillBeebe) [inline data/imaaquibali.json:58]: I'll get started on a script to add `EddieHub` everywhere. /s 

V small changes, LGTM for what it's worth. 👍

**Review by WillBeebe — APPROVED** [review]

**Review by SurajPratap10 — DISMISSED** [review]

**Review by mahfuztbt — DISMISSED** [review]

**Review by kumarsonsoff3 — DISMISSED** [review]
Looks good, Thanks! 🎉 

--
**Verified** that `aaquibdev.json` was first merged by @imaaquibali through #975. Hence it's his old profile.

File: data/imaaquibali.json (hunk 1 above)
Comment (imaaquibali) [reply data/imaaquibali.json]: Actually i the name of the community is not changed yet and my pervious profile is still visible should i make new pr for that or make changes in this pr

Comment (kumarsonsoff3) [reply data/imaaquibali.json]: This PR is being merged 3 Days ago, so now you need to create a new one for any other changes in your profile.

**Pull request merged**